### ISSUE TITLE
I have problem with termux-x11

### ISSUE THREAD

**Issue by namyunseong97** [Pipetto-crypto/Chroot-Docs#4]
![Screenshot_2022-10-10-14-48-21-1128443065](https://user-images.githubusercontent.com/102759766/194915334-f194c5d9-ee49-433d-8f6f-d57b1f232e95.png)
1. xwayland cannot be install with sudo.
![Screenshot_2022-10-10-14-47-25-0387172158](https://user-images.githubusercontent.com/102759766/194915406-5cb4485f-dd3d-480b-b323-e79300ff292c.png)
2. termux-x11 isnt included in termux repo.

**Comment by Pipetto-crypto** [comment]
Have you followed the guide in termux-x11 folder?

pkg install x11-repo
pkg install termux-x11 xwayland

Then inside the chroot, not in termux you need to do: sudo apt install xwayland

https://udroid-rc.gitbook.io/udroid-wiki/udroid-landing/quick-installation-and-usage/setting-up-gui/termux-x11

Follow this step by step up to the "Setting termux is done" line

**Comment by namyunseong97** [comment]
Yes. I installed with the guide. As you can see in image #2, It shows that "unable to locate package termux-x11" when I tried to install with "pkg install termux-x11 xwayland"

**Comment by Pipetto-crypto** [comment]
> Yes. I installed with the guide. As you can see in image #2, It shows that "unable to locate package termux-x11" when I tried to install with "pkg install termux-x11 xwayland"

Did you install the termux-x11 deb from the guide I posted. Can you run termux-x11 command and see if it finds it?

**Comment by namyunseong97** [comment]
Yes. I already installed termux-x11.deb with dpkg. but It can't be installed with "pkg install termux-x11"

**Comment by Pipetto-crypto** [comment]
> Yes. I already installed termux-x11.deb with dpkg. but It can't be installed with "pkg install termux-x11"

It seems they removed termux-x11 from the repo and now only way to get it is to install it with that deb. Try to run termux-x11 command and see if it's there

**Comment by namyunseong97** [comment]
ah. okay. thank you for your help. and have a nice day.
